src/org/omegat/gui/main/MainWindowNewMenu.java

```diff
@@ -41,13 +41,14 @@
  * 
  * @author Hiroshi Miura
  */
-public final class MainWindowBurgerMenu extends BaseMainWindowMenu
+public final class MainWindowNewMenu extends BaseMainWindowMenu
         implements ActionListener, MenuListener, IMainMenu {
 
     JMenu burgerMenu;
 
-    public MainWindowBurgerMenu(MainWindow mainWindow, MainWindowMenuHandler mainWindowMenuHandler) {
+    public MainWindowNewMenu(MainWindow mainWindow, MainWindowMenuHandler mainWindowMenuHandler) {
```

Comment:
If you add the possibility to add multiple variants like this, then I think the term "new UI" will become obsolete when there will be 3 or 4 implementations.
I would suggest to rename the classical class as MainWindowClassicMenu and find something else for the new UI

Reply:
Oh, yes, it is reason I put a name "BurgerMenu" in previous, but the class does not only consist a burger menu also consist with toobar interface. I don't want to change classic and standard menu class name.
The  "new UI" is strongly inspired from "newUI" feature of JetBrains IntelliJ IDEA 2023 version.
https://www.jetbrains.com/help/idea/new-ui.html#new_themes_icons_font

An idea to improve usability by avoiding modal dialog in semi frequent operations;  select target file.

Reply:
You could keep the name "burger", and we could refer to it as "burger style" in the documentation: I doubt anybody else will use this name for something else.
And rename the original class becomes necessary to avoid confusion (this class is _a_ menu, not anymore _the_ menu)

Reply:
revert name.